apps/challenges/github_sync_config.py

```diff
@@ -0,0 +1,37 @@
+# Fields from Challenge, ChallengePhase model to be considered for github_sync
+
+challenge_non_file_fields = [
```

Comment:
@savish28 is there a way we can use the serializer or model to generate this list of fields? Currently, if a new field gets added we'll have to add it here manually. It might happen that some forgets adding it here which would mean there is no sync happening for that field. If we can use models/serializer for this list it would be great

Reply:
@Ram81, we only wish to have only selected fields(that are editable from frontend) in the sync and not all fields, getting it from serializer would mean using all fields and that is undesirable because there are a lot of fields that are not in the challenge_config and might also be critical to share those.